gradle/versions.gradle

```diff
@@ -133,6 +133,7 @@ dependencyManagement {
     dependency 'org.miracl.milagro.amcl:milagro-crypto-java:0.4.0'
     dependency 'org.quartz-scheduler:quartz:2.3.2'
     dependency 'org.rocksdb:rocksdbjni:6.11.4'
+    dependency 'org.fusesource.leveldbjni:leveldbjni-all:1.8'
```

Comment:
Let's maybe use my custom build for version `1.18.3` for now. It should be far more stable than `1.8`. If I remember correctly we had some crashes and db corruptions on older version and thus created the custom build
```suggestion
    dependency 'org.ethereum:leveldbjni-all:1.18.3'
```

Reply:
We would also need to move the artifact from Bintray because of its shutdown